docs/man/git-lfs-ls-files.1.ronn

```diff
@@ -15,6 +15,10 @@ reference.  If no reference is given, scan the currently checked-out branch.
 * `-l` `--long`:
   Show the entire 64 character OID, instead of just first 10.
 
+* -d --debug:
```

Comment:
Can this be `--extra`?

Reply:
It could although my intention was to match the `git ls-files` option: git/git@8497421
Up to you. Either way is fine with me 🙂

Reply:
Ah, I did not know that `git ls-files` has `--debug`. I agree that we should continue to use that for consistency.